src/content/Auth/UserContext.tsx

```diff
@@ -0,0 +1,3 @@
+import {createContext} from 'react'
+
+export const UserContext = createContext(null);
```

Comment:
The context should have an interface that is passed into `createContext` to set its type